src/main/java/pl/wip/wipplatformbe/models/User.java

```diff
@@ -0,0 +1,52 @@
+package pl.wip.wipplatformbe.models;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import pl.wip.wipplatformbe.enums.Badge;
+import pl.wip.wipplatformbe.enums.Role;
+import pl.wip.wipplatformbe.enums.UserStatus;
+//import org.springframework.security:spring-security-crypto
+//import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Document
+public class User {
+
+    @Id
+    private String id;
+    private String psswrdHashed;
```

Comment:
Doesn't match the diagram. Study the UML diagrams more carefully.
![image](https://github.com/WIP-Work-In-Progress/wip-platform-be/assets/116646363/308bcf2c-9c67-47e2-b5c2-482dbec69259)